src/main/java/org/opentripplanner/netex/mapping/NetexMapper.java

```diff
@@ -356,11 +359,12 @@ private void mapFlexibleStopPlaces() {
     }
 
     private void mapDatedServiceJourneys() {
+        datedServiceJourneysBySjId.putAll(DatedServiceJourneyMapper.indexDSJBySJId(
+                currentNetexIndex.getDatedServiceJourneys()
+        ));
         tripCalendarBuilder.addDatedServiceJourneys(
             currentNetexIndex.getOperatingDayById(),
-            DatedServiceJourneyMapper.indexDSJBySJId(
-                currentNetexIndex.getDatedServiceJourneys()
-            )
+            datedServiceJourneysBySjId
```

Comment:
```suggestion
            dsjBySJId
```

Reply:
The NetexMapper.java is the root object for the entire mapping, it should not have any fields like `stationsByMultiModalStationRfs`, `stopTimesByNetexId` or `datedServiceJourneysBySjId`. Its responsibility is to orchestrate the mapping process. This is already broken (first by me), so I fixed it in a PR below, witch @JonathanBrynhagen can merge into his branch.